lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -2603,6 +2610,31 @@ def __init__(self, client):
             self.results['ansible_facts'] = {'docker_container': self.facts}
             self.results['container'] = self.facts
 
+    def wait_for_state(self, container_id, complete_states=None, wait_states=None, accept_removal=False):
+        delay = 1.0
```

Comment:
```
    def wait_for_state(self, container_id, complete_states=None, wait_states=None, accept_removal=False):
        delay = 1.0
```
we can implement this as a new module's parameter

Reply:
That's what I intend to do :)

Reply:
Perfect reader’s feedback on a column “You said exactly what i wanted to say but didn’t know how” , Thomas L. Freedman, Thank you for being late

It’s cool, feeling my progress:)